src/imarker_simple.cpp

```diff
@@ -121,9 +121,9 @@ void IMarkerSimple::make6DofMarker(const geometry_msgs::Pose& pose, double scale
   control.interaction_mode = InteractiveMarkerControl::MOVE_AXIS;
   int_marker_.controls.push_back(control);
 
-  control.orientation.w = 1;
+  control.orientation.w = 0.7071;
   control.orientation.x = 0;
-  control.orientation.y = 1;
+  control.orientation.y = 0.7071;
```

Comment:
Should this be -0.7071 .? I used this [website](https://quaternions.online/) for illustration

Reply:
It seems to work fine (screenshot attached). I guess somebody else would have noticed if there was an issue, since I didn't change the sign.

![interactive_marker_tool](https://user-images.githubusercontent.com/11284393/72183247-0f796d00-33b3-11ea-9549-8697677ec656.png)

Reply:
Note that the screenshot is after I dragged it around a bit